Machine_Learning/Supervised_Machine_Learning/ML_2_8_K-Nearest Neighbour Algorithm_(A).md

```diff
@@ -0,0 +1,13 @@
+## K Nearest Neighbours (A)
+
+In a ML task, we generally have two sorts of problems that are to be resolved possibly it very well may be 'Classification' or it tends to be a 'Regression' issue. 
+
+Classification is a process of arranging a given set of information into each unique class. Order can be executed on the two sorts of information organized just as unstructured or structured. Classes are frequently referred to as Labels or Targets which holds various classes. For instance, classifying different fruits. 
+
+Regression is a problem where our taget holds continuos or real values. Like prediction of age of an individual. 
+
+
+K-Nearest Neighbor classifier is one of the introductory classifiers, which every data science learner should be aware of.  KNN aims for pattern recognition tasks. 
+K-Nearest Neighbor also known as KNN is a supervised learning algorithm that can be used for regression as well as classification problems. Generally, it is used for classification in machine learning. 
+
+[Reference Audio](https://drive.google.com/drive/folders/1OObJrcyS45EDw5QMryfbebOCk9_IlCbA?usp=sharing)
```

Comment:
@sanjana-iyengar Please add a thumbnail like others have done and then hyperlink the audio file link with that. You can take help from the Demo folder, where we have discussed about how to add your audio or video files.

Reply:
Sure